src/tablet/tablet_impl.cc

```diff
@@ -4333,96 +4340,50 @@ void TabletImpl::RefreshTableInfo() {
+    if (!table) {
+        PDLOG(WARNING, "%s.%s not found", meta_db, meta_table);
+        return false;
     }
-}
-
-bool TabletImpl::GetClusterRouter() {
-    if (std::atomic_load_explicit(&sr_, std::memory_order_acquire)) return true;
-
-    PDLOG(INFO, "Init ClusterSDK in tablet server");
-    if (IsClusterMode()) {
-        ::openmldb::sdk::SQLRouterOptions copt;
-        copt.zk_cluster = zk_cluster_;
-        copt.zk_path = zk_path_;
-        auto sr = std::make_shared<::openmldb::sdk::SQLClusterRouter>(copt);
-        if (sr) {
-            if (!sr->Init()) {
-                PDLOG(WARNING, "Fail to init sql cluster router");
-                return false;
-            }
-
-            ::hybridse::sdk::Status status;
-            sr->ExecuteSQL("SET @@execute_mode='online';", &status);
-            if (!status.IsOK()) {
-                PDLOG(WARNING, "set online mode failed: %s ", status.msg);
-                return false;
-            } else {
-                DLOG(INFO) << "set online mode succeed";
-                std::atomic_store_explicit(&sr_, sr, std::memory_order_release);
-                return true;
-            }
-        } else {
-            PDLOG(WARNING, "create sql router failed");
-            return false;
-        }
-    } else {
-        PDLOG(ERROR, "Un-support standalone mode for now");
+    static ::hybridse::codec::RowView row_view(*(table->GetSchema()));
+    auto it = table->GetIterator();
+    if (!it) {
+        PDLOG(WARNING, "fail to get iterator. %s.%s", meta_db, meta_table);
         return false;
     }
+    it->SeekToFirst();
+    std::vector<::hybridse::vm::AggrTableInfo> table_infos;
+    while (it->Valid()) {
+        auto row = it->GetValue();
+        const char* str = nullptr;
+        uint32_t len = 0;
+        ::hybridse::vm::AggrTableInfo table_info;
+        row_view.GetValue(row.buf(), 0, &str, &len);
+        table_info.aggr_table.assign(str, len);
+        row_view.GetValue(row.buf(), 1, &str, &len);
+        table_info.aggr_db.assign(str, len);
+        row_view.GetValue(row.buf(), 2, &str, &len);
+        table_info.base_db.assign(str, len);
+        row_view.GetValue(row.buf(), 3, &str, &len);
+        table_info.base_table.assign(str, len);
+        row_view.GetValue(row.buf(), 4, &str, &len);
+        table_info.aggr_func.assign(str, len);
+        row_view.GetValue(row.buf(), 5, &str, &len);
+        table_info.aggr_col.assign(str, len);
+        row_view.GetValue(row.buf(), 6, &str, &len);
+        table_info.partition_cols.assign(str, len);
+        row_view.GetValue(row.buf(), 7, &str, &len);
+        table_info.order_by_col.assign(str, len);
+        row_view.GetValue(row.buf(), 8, &str, &len);
+        table_info.bucket_size.assign(str, len);
+
+        table_infos.emplace_back(std::move(table_info));
+        it->Next();
+    }
+    catalog_->RefreshAggrTables(table_infos);
+    DLOG(INFO) << "Refresh agg catalog in standalone mode (size = " << table_infos.size() << ")";
```

Comment:
`LOG(INFO) << "Refresh agg catalog in (size = " << table_infos.size() << ")";`

no need the `standalone mode` as they are merged.